CHANGELOG.md

```diff
@@ -16,6 +16,10 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
   [Dimitris Koutsogiorgas](https://github.com/dnkoutso)
   [#1698](https://github.com/CocoaPods/CocoaPods/issues/1698) 
 
+* Prevents need for .swift-version file in Objective-C pods  
+ +  [Austin Emmons](https://github.com/atreat)
```

Comment:
I get it now, looking at the rendered file makes more sense. I didn't release a double space at the end of a line in markdown is a way to force a newline.